packages/grafana-ui/package.json

```diff
@@ -56,23 +55,21 @@
     "rc-drawer": "4.4.0",
     "rc-slider": "9.6.4",
     "rc-time-picker": "^3.7.3",
-    "react": "17.0.1",
     "react-beautiful-dnd": "13.1.0",
     "react-calendar": "2.19.2",
     "react-colorful": "5.1.2",
     "react-custom-scrollbars-2": "4.4.0",
-    "react-dom": "17.0.1",
     "react-dropzone": "11.3.4",
     "react-highlight-words": "0.17.0",
-    "react-hook-form": "7.5.3",
     "react-inlinesvg": "2.3.0",
     "react-popper": "2.2.4",
-    "react-router-dom": "^5.2.0",
     "react-select": "4.3.0",
     "react-table": "7.7.0",
+    "react-test-renderer": "17.0.1",
```

Comment:
Why is this a dependency? I assume this is only needed for testing.

Reply:
You're right, will fix